src/test/java/com/katelocate/menugenerator/RecipeRepositoryTests.java

```diff
@@ -95,7 +105,12 @@ void shouldGetRecipeById() {
                 .get("/api/recipes/0")
                 .then()
                 .statusCode(200)
-                .body(equalTo("{\"id\":0,\"title\":\"Breakfast\",\"recipeType\":\"BREAKFAST\",\"body\":\"Scramble eggs\"}"));
+                .body(equalTo(
+                        """
+                            {"id":0,"title":"Breakfast","recipeType":"BREAKFAST","body":"Scramble eggs"}
+                        """
```

Comment:
Nice, looks much better. But I'd still try to find a way to reuse some constants